### PR TITLE
Feat/lesq 143/new style video

### PR DESCRIPTION
**Pull request by kimon-satan**
# I want to see the new style video in lesson overview

## Acceptance Criteria:

- [x]  No share with pupils (share functionality coming later)
- [x]  Border around video
- [x]  Drop down for transcript (transcript text doesn’t go full width)
- [x]  Blue div above
- [x]  Signed button (as before but slightly different styling)
- [x]  Same column width that the lesson details uses to help improve readability
- [x]  Mobile view
- [x]  Accessibility checks

## How to test

1. Go to https://deploy-preview-1778--oak-web-application.netlify.thenational.academy/beta/teachers/programmes/computing-secondary-ks3/units/impact-of-technology-collaborating-online-respectfully-35d0/lessons/account-security-68rkee
2. Scroll to Video

## Screenshots

How it used to look (delete if n/a):

image

How it should now look:

image

## Checklist

- [ ] Added or updated tests where appropriate
- [ ] Manually tested across browsers / devices
- [ ] Considered impact on accessibility
- [ ] Design sign-off
- [ ] Approved by product owner

### CODE REVIEW

File: src/pages/beta/[viewType]/programmes/[programmeSlug]/units/[unitSlug]/lessons/[lessonSlug].tsx [hunk 1]
```diff
@@ -244,7 +246,7 @@ const LessonOverviewPage: NextPage<LessonOverviewPageProps> = ({
           </Box>
         ) : (
           <>
-            <Grid $pt={[48]}>
+            <Grid id="test" $pt={[48]}>
```
Comment (mantagen) [inline src/pages/beta/[viewType]/programmes/[programmeSlug]/units/[unitSlug]/lessons/[lessonSlug].tsx:249]: Is this used anywhere? If so, could we give it a unique id?

File: src/pages/beta/[viewType]/programmes/[programmeSlug]/units/[unitSlug]/lessons/[lessonSlug].tsx [hunk 2]
```diff
@@ -277,14 +279,16 @@ const LessonOverviewPage: NextPage<LessonOverviewPageProps> = ({
                   </LessonItemContainer>
 
                   {videoMuxPlaybackId && (
-                    <ExpandingContainer {...curriculumData} title={"Video"}>
+                    // <ExpandingContainer {...curriculumData} title={"Video"}>
```
Comment (mantagen) [inline src/pages/beta/[viewType]/programmes/[programmeSlug]/units/[unitSlug]/lessons/[lessonSlug].tsx:282]: Can delete?

**Review by mantagen — COMMENTED** [review]
Looks so much better!

File: src/pages/beta/[viewType]/programmes/[programmeSlug]/units/[unitSlug]/lessons/[lessonSlug].tsx (hunk 1 above)
Comment (kimon-satan) [reply src/pages/beta/[viewType]/programmes/[programmeSlug]/units/[unitSlug]/lessons/[lessonSlug].tsx:249]: whoops. I'll remove that

File: src/pages/beta/[viewType]/programmes/[programmeSlug]/units/[unitSlug]/lessons/[lessonSlug].tsx (hunk 2 above)
Comment (kimon-satan) [reply src/pages/beta/[viewType]/programmes/[programmeSlug]/units/[unitSlug]/lessons/[lessonSlug].tsx]: whoops yep.

**Review by jack-skerrett-oak — APPROVED** [review]
Looks good.

One odd quirk that I don't think would stop it from being merged is when using a screen reader and selecting one of the "show x" buttons, you have to re-tab onto the button after selecting it. Just feels a little clunky, but you don't get stuck so I am happy to approve!

**Pull request merged**